sema/subyt/sinks.py

```
@@ -9,17 +9,18 @@
+    out_path = Path(path_name)
     if not force_output:
-        assert not os.path.isfile(
-            file_path
-        ), f"File to write '{file_path}' already exists"
-    parent_path = Path(file_path).parent.absolute()
-    if not os.path.exists(parent_path):
-        os.makedirs(parent_path)
-    assert os.access(
-        parent_path, os.W_OK
-    ), f"Can not write to folder '{parent_path}' for creating new files"
+        assert not out_path.exists(), (
```

Ruff (0.8.2)

Use of `assert` detected

(S101)